crates/bitwarden/src/auth/tde.rs

```diff
@@ -0,0 +1,52 @@
+use base64::{engine::general_purpose::STANDARD, Engine};
+use bitwarden_crypto::{
+    AsymmetricEncString, AsymmetricPublicCryptoKey, DeviceKey, EncString, SymmetricCryptoKey,
+    TrustDeviceResponse, UserKey,
+};
+
+use crate::{error::Result, Client};
+
+pub(super) fn make_register_tde_keys(
```

Comment:
Should we lift up the inline documentation here?  I think the code is pretty self-explanatory but it's good to describe what's happening for consumers.